scripts/jenkins/jobs-obs/openstack-rpm-packaging-update.yaml

```diff
@@ -32,3 +32,12 @@
           osc service dr
           osc addremove
           osc commit -m "Automatically updated rpm-packaging-openstack via Jenkins"
+
+          # also create links for new packages
+          echo "creating missing links ..."
+          for x in *-SLE_12_SP1.spec; do
+              pkgname=(${x//-SLE_12_SP1.spec/ })
```

Comment:
Right. Thanks for the hint. Fixed in https://github.com/SUSE-Cloud/automation/pull/1013/commits/83e8c4f6cb5761d8b6213d984f9d8b0e38225706